### PR TITLE
fix: request control of vehicles to fix them

### PR DESCRIPTION
**Pull request by NoobKeksTV**
sometimes a player cant fix a vehicle cause he isnt the one, which is allowed to control it. So i changed it to request control of the vehicle first to make sure we can repair it.

### CODE REVIEW

File: scripts/menu/client/cl_menu.lua [hunk 1]
```diff
@@ -234,7 +234,8 @@ local function toggleFreecam(enabled)
   FreezeEntityPosition(ped, enabled)
   
   if enabled then
-    freecamVeh = GetVehiclePedIsIn(ped, false)
+    freecamVeh = Get
```
Comment (GoatG33k) [inline scripts/menu/client/cl_menu.lua]: Invalid? lol

Comment (NoobKeksTV) [reply scripts/menu/client/cl_menu.lua]: i didnt even change stuff there but ok. lol

**Review by NoobKeksTV — COMMENTED** [review]
fixxed some mistake

File: scripts/menu/client/cl_menu.lua [hunk 2]
```diff
@@ -235,6 +235,7 @@ local function toggleFreecam(enabled)
   
   if enabled then
     freecamVeh = GetVehiclePedIsIn(ped, false)
+    PedIsIn(ped, false)
```
Comment (GoatG33k) [inline scripts/menu/client/cl_menu.lua:238]: lol... did you even test before committing this?

**Review by GoatG33k — CHANGES_REQUESTED** [review]

File: scripts/menu/client/cl_menu.lua (hunk 2 above)
Comment (NoobKeksTV) [reply scripts/menu/client/cl_menu.lua:238]: i didnt even change that stuff wtf.

**Pull request closed without merging**